documentation/staging/content/faq/external-clients.md

```diff
@@ -366,6 +381,129 @@ To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA
   * Also apply patch 30656708 on each target WebLogic Server for versions 12.2.1.4 (PS4) or earlier.
 
+#### Configuring WebLogic Server Affinity Load Balancing Algorithm
+
+When providing external clients access to EJB and JMS resources using load balancer tunneling, it is recommended to configure a server affinity load balancing algorithm for the cluster in which the resources are targeted.  Using a server affinity based algorithm turns off load balancing between external Java clients and server instances causing method calls to 'stick' to a server instance to which the external client is communicating with.  For more information see [Load Balancing for EJBs and RMI Objects](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/clust/load_balancing.html#GUID-2470EEE9-F6F9-44EF-BA54-671728E93DE6) in _Load Balancing in a Cluster_.
+
+Here are steps for enabling server affinity in a cluster, named 'cluster-1', for a domain source type __Domain in PV__:
+
+```
+How to implement Server Affinity in Cluster: 
+=========================================== 
+1. Go to  admin console 
+2. Expand the Clusters link on the left panel 
+3. Click on 'cluster-1' link 
+    i.   look on the right panel. 
+    ii.  look for Default Load Algorithm property 
+    iii. change the value to "RoundRobinAffinity" instead of "RoundRobin". 
+4. Restart the domain/weblogic server instances. 
+``` 
+
+Here is a snippet of offline WLST code for enabling server affinity in a cluster 'cluster-1':
+
+```javascript
+clusterName = "cluster-1"
+cd('/Clusters/%s' % clusterName)
+set('DefaultLoadAlgorithm', 'round-robin-affinity')
+```
+Here is a snippet of WDT model YAML file configuration for enabling server affinity in a cluster 'cluster-1':
+
+```yaml
+topology:
+    Cluster:
+        'cluster-1':
+            DynamicServers:
+                ServerTemplate:  'cluster-1-template'
+                ServerNamePrefix: 'managed-server'
+                DynamicClusterSize: '5'
+                MaxDynamicClusterSize: '5'
+                MinDynamicClusterSize: '0'
+                CalculatedListenPorts: false
+            DefaultLoadAlgorithm: 'round-robin-affinity'
+```
+
+#### Configuring External Listen Address for WebLogic Default Channel
+
+WebLogic Server provides an External Listen Address/DNS Name for use in RMI stubs to allow external clients to connect to the server through a Kubernetes external service name. For more information on 'External Listen Address' (i.e. ExternalDNSName attribute) on the ServerMBean, see ExternalDNSName attribute in [ ServerMBean reference ](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/wlmbr/core/index.html).
```

Comment:
"RMI stub" is sort of an internal/opaque concept to customers -- and the external LA feature was not created specifically for "external kubernetes services".

how about rewording to "WebLogic Server provides an External Listen Address/DNS Name feature to enable external JMS and EJB clients to properly reestablish and/or load balance connections to servers with listen addresses that aren't directly available in the client's DNS (such as through a firewall or a load-balancer, or between different namespaces in Kubernetes).